traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/router/HTTPRouteResult.java

```diff
@@ -60,6 +61,10 @@ public void setUrl(final URL url) {
 		return deliveryServices;
 	}
 
+	public String getDeliveryServicesLogString() {
+		return this.getDeliveryServices().stream().map(DeliveryService::getId).collect(Collectors.joining("|"));
```

Comment:
Is this due to `CLIENT_STEERING` delivery services? We want to log the target DS XML IDs joined with `|` instead of logging the XML ID of the `CLIENT_STEERING` DS?

Reply:
Just tested with a `CLIENT_STEERING` DS and as of right now it only logs the target. L65 was originally added only because internally the `xml_id`s of an HTTP request are stored internally as a `List<>` object. Would you prefer only having the `CLIENT_STEERING` DS, or a combination of the two?